JupyterKernel/xSlicerInterpreter.cxx

```diff
@@ -68,6 +68,11 @@ xjson xSlicerInterpreter::execute_request_impl(int execution_counter,
     QVariant executeResult = pythonManager->executeString(qscode.left(qscode.length() - displayCommand.length()));
     pub_data["image/png"] = execute_display_command();
   }
+  else if (qscode.endsWith("__toggle_kernel_debug()"))
+  {
+    m_print_debug_output = !m_print_debug_output;
+    std::cout << "Toggled debug output: " << (m_print_debug_output ? "on" : "off") << std::endl;
```

Comment:
Re-running the code would randomly enable/disable debug output. Could you change this so that you can set if you want debug output or not? For example __kernel_debug_enable()/__kernel_debug_disable() or __kernel_debug_enable(True)/__kernel_debug_enable(False.)

Reply:
Ok, updated.